.gitignore

```diff
@@ -2,3 +2,4 @@ cache/
 node_modules/
 public/
 build/
+vendor/
```

Comment:
This is obsolete

Reply:
Thanks for spotting that. Something I overlooked when removing the PHP code.